ISLE/isleapp.cpp

```diff
@@ -400,7 +400,7 @@ SDL_AppResult SDL_AppEvent(void* appstate, SDL_Event* event)
 		case WM_ISLE_SETCURSOR:
 			g_isle->SetupCursor((Cursor) (uintptr_t) event->user.data1);
 			break;
-		case WM_QUIT:
+		case WM_CLOSE:
```

Comment:
Reading this [stackoverflow answer](https://stackoverflow.com/questions/3155782/what-is-the-difference-between-wm-quit-wm-close-and-wm-destroy-in-a-windows-pr), I'd emit `WM_QUIT` inside lego1.

But either ways, this approach looks good to me.